tests/format/flow/no-semi/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -77,20 +77,22 @@ semi: false
                                                                                 | printWidth
 =====================================input======================================
 class A {
-  +one = val();
-  +two = val();
+  +one = function() {};
+  -two = val();
   static +three = val();
   +#privOne = val();
   static +#privTwo = val();
+  [computed] = val();
 }
```

Comment:
I'm sorry, didn't make it clear, I mean `+[computed] = 1`, is it valid?

Reply:
Ah right, I added the variance sigil in 87a72d20a  
Note though that semi is already inserted for that case in the current Prettier stable, since semi is always added before computed class properties.